qoffer-infrastructure/src/main/groovy/life/qbic/portal/qoffer2/web/views/CreateCustomerView.groovy

```diff
@@ -142,4 +142,60 @@ class CreateCustomerView extends FormLayout {
     }
 
 
+    private boolean allValuesValid = {
```

Comment:
Is there a reason for defining a closure here? If not, I would favor a common method.

Reply:
I was also a little confused by the closure here and would also favor a common method

Reply:
not really. I had a field at first that was set with every error occurrence or non-occurence and figured at some point that I could also read the value on the fly.

Reply:
will fix it